src/components/AbstractSelectableComponent.js

```diff
@@ -108,8 +110,12 @@ export default class AbstractSelectableComponent extends React.Component {
     updateViewInfo(this.viewInfo);
   }
 
+  onWebGLInitialized(gl) {
+    this.setState({ gl });
+  }
+
```

Comment:
This function is needed for passing in the webgl context of the canvas to the Texture creation step